### PR TITLE
Add io.github.TransmissionRemoteGtk

### PR DESCRIPTION
**Pull request by hfiguiere**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ ] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** I am not. The release upstream were tagged by @TingPing 
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge). No, I used the one set upstream. Note: they may not meet the flathub criterias. @TingPing was the committer
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)* not yet. Will submit oars patch, but the screenshot is because it's 403 forbidden upstream.

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by hfiguiere** [comment]
bot, build io.github.TransmissionRemoteGtk

**Comment by hfiguiere** [comment]
The icon wasn't copied so the app couldn't find it.

bot, build io.github.TransmissionRemoteGtk

File: io.github.TransmissionRemoteGtk.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+    "id": "io.github.TransmissionRemoteGtk",
+    "runtime": "org.gnome.Platform",
+    "sdk": "org.gnome.Sdk//41",
```
Comment (barthalion) [inline io.github.TransmissionRemoteGtk.json:4]:
```suggestion
    "sdk": "org.gnome.Sdk",
```

**Comment by barthalion** [comment]
Looks good to me, but I'd appreciate @TingPing's approval.

**Comment by barthalion** [comment]
Commits since release don't look controversial, so maybe it would also make sense to build it from master.

File: io.github.TransmissionRemoteGtk.json (hunk 1 above)
Comment (hfiguiere) [reply io.github.TransmissionRemoteGtk.json:4]: gotta fix my generator :-/

Comment (hfiguiere) [reply io.github.TransmissionRemoteGtk.json]: fixed now

**Comment by hfiguiere** [comment]
> Commits since release don't look controversial, so maybe it would also make sense to build it from master.

I'd stick to release. I synced the screenshot with master and the patch to appdata is already appstream

**Pull request closed without merging**